fuseimg/datasets/knight.py

```diff
@@ -245,12 +242,7 @@ def dataset(
         cache_dir: str = "cache",
         split: dict = None,
```

Comment:
How should a user generate a split?
I think that it's better to get sample_ids as the other examples and to include all samples if it was set to None.

Reply:
there is a splits pkl file which contains 5 different splits which are necessary to compare results.

Reply:
It's ok to use it for the example you are implementing.
But the dataset implementation needs to be more general, so we shouldn't ask the users to create their own split file to use it.
And if we do, you need to carefully explain the structure and how to create a new one.